ru/lessons/basics/mix.md

```diff
@@ -125,13 +125,13 @@ $ mix deps.get
 
 ## Окружения
 
-Mix, как и Bundler, поддерживает различные окружения. "Из коробки" mix работает с тремя окружениями:
+Mix, как и Bundler, поддерживает различные окружения. После установки Mix работает с тремя окружениями:
 
 + `:dev` — Окружение по умолчанию.
 + `:test` — Используется в `mix test`. Будет рассмотрено подробнее в следующем уроке.
-+ `:prod` — Используется при запуске приложения в "боевом" окружении.
++ `:prod` — Используется при запуске приложения в рабочем окружении.
 
-Текущее окружение можно получить с помощью `Mix.env`. При запуске любой команды его можно задать с помощью переменной окружения `MIX_ENV`:
+Текущее окружение можно получить с помощью `Mix.env`. При запуске любой команды его можно установить с помощью переменной окружения `MIX_ENV`:
```

Comment:
`Текущее окружение` doesn't sound right here.

Reply:
Задать?

Reply:
I believe it could be something like this

```
Текущий режим окружения можно получить с помощью функции `Mix.env/0`. При запуске любой команды его можно установить с помощью переменной окружения `MIX_ENV`:
```